app/lib/headless-browser.js

```diff
@@ -0,0 +1,39 @@
+const puppeteer = require('puppeteer');
+
+const args = ['--no-startup-window'];
+const userDataDir = './chromium-cache';
+
+/**
+ * This class approach makes it easy to open multiple browser instances with
+ * different arguments in case that is ever required.
```

Comment:
In OpenClinica's fork we actually do this with custom 'headless' API endpoints that serve to import records, run validation on them, and add comments to errors. Those endpoints use a different headless browser config to optimize performance for that purpose (without CSS e.g.)